### PR TITLE
Fixed link to RHBA-2018:2335

### PR DESCRIPTION
**Pull request by ghost**
Follow up to https://github.com/openshift/openshift-docs/issues/11572.

### CODE REVIEW

**Comment by tmorriso-rh** [comment]
@ahardin-rh PTAL. Thanks.

Cc: @vikram-redhat

File: release_notes/ocp_3_9_release_notes.adoc [hunk 1]
```diff
@@ -2165,7 +2165,7 @@ Issued: 2018-08-09
 
 {product-title} release 3.9.40 is now available. The list of packages and
 bug fixes included in the update are documented in the
-link:https://access.redhat.com/errata/RHSA-2018:2335[RHBA-2018:2335] advisory.
+link:https://access.redhat.com/errata/RHSA-2018:2335[RHSA-2018:2335] advisory.
```
Comment (ahardin-rh) [inline release_notes/ocp_3_9_release_notes.adoc]: @tmorriso-rh The link should be:


    link:https://access.redhat.com/errata/RHBA-2018:2335[RHBA-2018:2335]

The RHSA path leads to a 404 error.

**Comment by tmorriso-rh** [comment]
@ahardin-rh Fixed. Can you take another look? Thanks.

**Comment by ahardin-rh** [comment]
@tmorriso-rh LGTM! :+1: :shipit:

**Pull request merged**